crates/ignore/src/default_types.rs

```diff
@@ -24,6 +24,7 @@ pub const DEFAULT_TYPES: &[(&[&str], &[&str])] = &[
     (&["ats"], &["*.ats", "*.dats", "*.sats", "*.hats"]),
     (&["avro"], &["*.avdl", "*.avpr", "*.avsc"]),
     (&["awk"], &["*.awk"]),
+    (&["batchfile"], &["*.bat"]),
```

Comment:
Would `bat` be better here?

Reply:
May I suggest `bat` in addition to `batchfile`? As these are commonly called "batch files", even by Windows.

Reply:
I suppose that's fine.

Reply:
Well, I'd prefer `batch` and `bat`, not `batchfile`. `file` seems redundant in this case and no other file type uses `file` in the name.

Reply:
`bat` and `batch` it is then.